tests/integration/docusaurus/connecting_to_your_data/database/snowflake_python_example.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from ruamel import yaml
+
+import great_expectations as ge
+
+from .util import load_data_into_database
+
+sfAccount = os.environ.get("SNOWFLAKE_ACCOUNT")
+sfUser = os.environ.get("SNOWFLAKE_USER")
+sfPswd = os.environ.get("SNOWFLAKE_PW")
+sfDatabase = os.environ.get("SNOWFLAKE_DATABASE")
+sfSchema = os.environ.get("SNOWFLAKE_SCHEMA")
+sfWarehouse = os.environ.get("SNOWFLAKE_WAREHOUSE")
+
+
+CONNECTION_STRING = f"snowflake://{sfUser}:{sfPswd}@{sfAccount}/{sfDatabase}/{sfSchema}?warehouse={sfWarehouse}"
+load_data_into_database(
+    "taxi_data",
+    "/Users/work/Development/great_expectations/integration/fixtures/data/reports/yellow_tripdata_sample_2019-01.csv",
```

Comment:
A tiny oversight - this is an absolute path on your machine. 😄

Reply:
ha! how could i have missed this